starter-code/scripts/models/article.js

```diff
@@ -99,8 +104,8 @@
 
   Article.truncateTable = function() {
     webDB.execute(
-      // TODO: Use correct SQL syntax to delete all records from the articles table.
-      'DELETE ...;' // <----finish the command here, inside the quotes.
+      // DONE: Use correct SQL syntax to delete all records from the articles table.
+      'DELETE * FROM articleTable;' // <----finish the command here, inside the quotes.
```

Comment:
For truncating an entire table, the syntax is "DELETE FROM articleTable" (no asterisk).